diff --git a/scheduler/conf.c b/scheduler/conf.c
index 758b822..6ab96ad 100644
--- a/scheduler/conf.c
+++ b/scheduler/conf.c
@@ -241,6 +241,11 @@ cupsdCheckPermissions(
   struct stat	fileinfo;		/* Stat buffer */
   int		is_symlink;		/* Is "filename" a symlink? */
 
+  /*
+   * Do not fix permissions of any files in OS maintained locations
+   */
+  if (strncmp(filename, "/usr/share", strlen("/usr/share")) == 0)
+    return (0);
 
  /*
   * Prepend the given root to the filename before testing it...
